<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(93, 0, 1);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(139, 103, 103);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ebrima&quot;;
color: rgb(170, 0, 0)</string>
   </property>
   <property name="text">
    <string>Статистика о победах</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>641</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 170, 170);
color: rgb(85, 0, 0);
font: 15pt &quot;MS Sans Serif&quot;;</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>206</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="max_victory">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 0, 0);
background-color: rgb(193, 170, 170);</string>
   </property>
   <property name="text">
    <string> Всего побед: -</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="best_time">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>330</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 0, 0);
background-color: rgb(193, 170, 170);</string>
   </property>
   <property name="text">
    <string> Лучшее время: -</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="update_button">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>720</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(93, 0, 1);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(139, 103, 103);</string>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
